### PR TITLE
fix: second click of network won't close network panel

### PR DESCRIPTION
**Pull request by asterwyx**
Network panel used to be controlled by dss-network-plugin, click
button simply show the panel. Let panel controlled by dde-session-shell.
dss-network-plugin just return its content. Use a fake popup instead of
a popup window.

### CODE REVIEW

File: src/global_util/modules_loader.cpp [hunk 1]
```diff
@@ -19,6 +19,9 @@ const QString ModulesDir = "/usr/lib/dde-session-shell/modules";
 ModulesLoader::ModulesLoader(QObject *parent)
     : QThread(parent)
 {
+    addModulePath(ModulesDir);
```
Comment (18202781743) [inline src/global_util/modules_loader.cpp]: 是不是应该优先找localDir的module

File: src/app/dde-lock.cpp [hunk 2]
```diff
@@ -97,9 +93,22 @@ int main(int argc, char *argv[])
...
+    cmdParser.addOption(modulePath);
+
     cmdParser.process(*app);
 
+    dss::module::ModulesLoader *modulesLoader = &dss::module::ModulesLoader::instance();
+
+    if (cmdParser.isSet(modulePath)) {
+        QString modulePathValue = cmdParser.value(modulePath);
+        QStringList paths = modulePathValue.split(":");
+        modulesLoader->clearModulePath();
```
Comment (18202781743) [inline src/app/dde-lock.cpp]: 为什么需要先clear,应该有个添加的优先级就可以了，或者提供set接口，而不是两个接口组合起来用

File: src/app/dde-lock.cpp [hunk 3]
```diff
@@ -97,9 +93,22 @@ int main(int argc, char *argv[])
     QCommandLineOption lockscreen(QStringList() << "l" << "lockscreen", "show lock screen");
     cmdParser.addOption(lockscreen);
 
-    QStringList xddd = app->arguments();
+    QCommandLineOption modulePath("p", "paths to load modules in debug mode, separated by semicolon.", "paths", QString());
```
Comment (zccrs) [inline src/app/dde-lock.cpp]: 段落开头首字母大写

**Review by zccrs — CHANGES_REQUESTED** [review]

**Comment by 18202781743** [comment]
+1

File: src/widgets/controlwidget.h [hunk 4]
```diff
@@ -44,27 +44,27 @@ class RoundPopupWidget;
...
         : DFloatingButton(iconType, parent) {
         installEventFilter(this);
     }
-    explicit FlotingButton(const QString &text, QWidget *parent = nullptr)
+    explicit FloatingButton(const QString &text, QWidget *parent = nullptr)
         : DFloatingButton(text, parent) {
         installEventFilter(this);
     }
-    FlotingButton(const QIcon& icon, const QString &text = QString(), QWidget *parent = nullptr)
+    FloatingButton(const QIcon& icon, const QString &text = QString(), QWidget *parent = nullptr)
```
Comment (BLumia) [inline src/widgets/controlwidget.h:67]: cppcheck: Class 'FloatingButton' has a constructor with 1 argument that is not explicit. Such constructors should in general be explicit for type safety reasons. Using the explicit keyword in the constructor means some mistakes when using the class can be avoided.

Comment (asterwyx) [reply src/widgets/controlwidget.h:67]: 我这次的提交只改了拼写的问题，这种错误是不是应该分一次提交出来。

Comment (BLumia) [reply src/widgets/controlwidget.h:67]: 可以，那就不在此 PR 处理也行

**Review by BLumia — DISMISSED** [review]

**Comment by asterwyx** [comment]
网络插件的通知功能异常，需要追加提交。

**Review by zccrs — DISMISSED** [review]

**Comment by zccrs** [comment]
/+1

File: src/app/dde-lock.cpp [hunk 5]
```diff
@@ -30,6 +30,8 @@ DWIDGET_USE_NAMESPACE
 
 int main(int argc, char *argv[])
 {
+    DLogManager::registerConsoleAppender();
+    DLogManager::registerFileAppender();
```
Comment (18202781743) [inline src/app/dde-lock.cpp]: 这个file appender会根据applicationName来组织log的路径，这样改就读不到正确的了，并且会提前构造qapplication,

**Review by BLumia — APPROVED** [review]

**Pull request merged**